File: frontend/src/constants/legal_act_object.ts
import { Dayjs } from "dayjs";

export type legal_act_object = {
  
  id: number;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
  id_act: number;
  id_object: number;
};

File: backend/Domain/Entities/legal_act_object.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class legal_act_object
    {
        public int id { get; set; }
		public DateTime? created_at { get; set; }
		public DateTime? updated_at { get; set; }
		public int? created_by { get; set; }
		public int? updated_by { get; set; }
		public int id_act { get; set; }
		public int id_object { get; set; }
		
    }
}